jgrapht-core/src/main/java/org/jgrapht/alg/shortestpath/GraphMeasurer.java

```diff
@@ -1,5 +1,5 @@
 /*
- * (C) Copyright 2017-2018, by Joris Kinable and Contributors.
+ * (C) Copyright 2017-2018, by Joris Kinable, Alexandru Valeanu and Contributors.
```

Comment:
Please leave this as is: "(C) Copyright 2017-2018, by Joris Kinable and Contributors."
(the first/main author maintains the copyright, additional authors are captured under 'contributors' and in the @author list